### ISSUE TITLE
Using Google Cloud Storage API prevents downloading from succeeding for Chinese users

### ISSUE THREAD

**Issue by gbraad** [openshift/origin#17525]
Using Google Cloud Storage API prevents downloading from succeeding for Chinese users. This seems to be introduced since the release of v3.7.0.

##### Steps To Reproduce
1. step off the plane in China
2. take out your notebook or phone and open https://github.com/openshift/origin/releases/tag/v3.7.0
3. open the link: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/
4. try to download any of the links in the GCS browser:
  * https://storage.googleapis.com/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz
  * https://storage.googleapis.com/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/openshift-origin-client-tools-v3.7.0-7ed6862-mac.zip
  * https://storage.googleapis.com/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/openshift-origin-client-tools-v3.7.0-7ed6862-windows.zip
  * ...

##### Current Result
Downloads will timeout in the browser

##### Expected Result
Download will succeed

##### Additional Information
```
$ wget https://storage.googleapis.com/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz
--2017-11-29 19:29:28--  https://storage.googleapis.com/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/
artifacts/zips/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz
Resolving storage.googleapis.com (storage.googleapis.com)... 172.217.24.16, 2404:6800:4012:1::2010
Connecting to storage.googleapis.com (storage.googleapis.com)|172.217.24.16|:443... connected.
GnuTLS: Error in the pull function.
Unable to establish SSL connection.
```

Note: this means downloads of the client will need to happen over VPN or proxies.

**Comment by gbraad** [comment]
![image](https://user-images.githubusercontent.com/1894/33412948-977dc524-d5c9-11e7-8e0b-3dc3295266eb.png)

![image](https://user-images.githubusercontent.com/1894/33412967-b5c1a384-d5c9-11e7-9528-64ce6a239f53.png)

**Comment by eliu** [comment]
@gbraad As you requested, the same result happens to me. Timeout.. Actually, 99% of the Google services are NOT available in China, to be honest~ GFW...

**Comment by eliu** [comment]
@gbraad just get you updated, Trying to download any files from https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/branch-logs/origin/v3.7.0/builds/test_branch_origin_cross/1032/artifacts/zips/ does not have any problem in China. But the links you provideed in OP broke.

**Comment by gbraad** [comment]
@eliu currently they also work for me from Beijing, but this is intermittent at best.

**Comment by eliu** [comment]
@gbraad yes, not too stable yet. First download failed with timeout, then successfully got the file at some time else.

**Comment by gbraad** [comment]
Originally they weren't published on Github as can be seen here: https://github.com/openshift/origin/issues/17527#issuecomment-348102372 Since they have been published for the 3.7.0 release at: https://github.com/openshift/origin/releases/tag/v3.7.0 this issue can be closed. Thanks

**Comment by gbraad** [comment]
In light of the comment: https://github.com/minishift/minishift/issues/1775#issuecomment-348516242 I rather re-open the issue.
